### PR TITLE
Add Mcp960x drivers, samples, datasheets

### PR DESCRIPTION
**Pull request by adrianstevens**

### CODE REVIEW

File: Source/Mcp9601.cs [hunk 1]
```diff
@@ -0,0 +1,8 @@
+﻿using System;
+
+public class Class1
```
Comment (jorgedevs) [inline Source/Mcp9601.cs:3]: maybe rename this class? :)

File: Source/Mcp9600.cs [hunk 2]
```diff
@@ -0,0 +1,9 @@
+﻿namespace Meadow.Foundation.Sensors.Temperature
+{
+    public partial class Mcp9600 : Mcp960x
```
Comment (jorgedevs) [inline Source/Mcp9600.cs:3]: This class seems in the wrong place, its in the repo root.

File: Source/Meadow.Foundation.Peripherals/Sensors.Temperature.Mcp960x/Driver/Mcp960x.cs [hunk 3]
```diff
@@ -0,0 +1,293 @@
...
+                conditions.TemperatureCold = new Units.Temperature(ReadTemperatureCold(), Units.Temperature.UnitType.Celsius);
+
+                return conditions;
+            });
+        }
+
+        /// <summary>
+        /// Reads the hot junction temperature from the MCP960x
+        /// </summary>
+        /// <returns>The hot junction temperature in degrees Celsius.</returns>
```
Comment (jorgedevs) [inline Source/Meadow.Foundation.Peripherals/Sensors.Temperature.Mcp960x/Driver/Mcp960x.cs:73]: Some comments have periods and some dont, want to consolidate while we're here?

**Review by jorgedevs — CHANGES_REQUESTED** [review]

File: Source/Mcp9601.cs (hunk 1 above)
Comment (jorgedevs) [reply Source/Mcp9601.cs:3]: This class is also in the repo's root.

File: Source/Meadow.Foundation.Peripherals/Sensors.Temperature.Mcp960x/Driver/Mcp960x.cs (hunk 3 above)
Comment (adrianstevens) [reply Source/Meadow.Foundation.Peripherals/Sensors.Temperature.Mcp960x/Driver/Mcp960x.cs:73]: yeah I'll do a quick pass .... chat gpt adds em

**Review by jorgedevs — APPROVED** [review]

**Pull request merged**